client/post-editor/editor-ground-control/index.jsx

```diff
@@ -284,4 +284,23 @@ export class EditorGroundControl extends PureComponent {
 	}
 }
 
-export default localize( EditorGroundControl );
+const mapDispatchToProps = dispatch => ( {
+	recordPreviewButtonClick: () =>
+		dispatch(
+			composeAnalytics(
+				recordTracksEvent(
+					`calypso_editor_${ postUtils.isPage( page ) ? 'page' : 'post' }_preview_button_click`
```

Comment:
do we want to address CPTs here or are we fine tracking them as `post`?

Reply:
I think we're good on tracking custom post types as `post`. Thanks for double checking!

Reply:
Our publish event, `calypso_editor_post_publish`, handles the custom post types the same way. Let's change this in a separate PR if we choose to handle custom post types differently.